back/src/pods/storage/api/index.ts

```diff
@@ -0,0 +1,5 @@
+import { envConstants } from 'core/env.constants';
+import * as mockApi from './storage.mock';
+import * as api from './storage.api';
+
+export const storageApi = envConstants.isApiMock ? mockApi : api;
```

Comment:
Mmm... thinking loud, since all functions imported are grouped into storageApi, could we create an interface enclosing all the functions signatures and hence forcing the function name matches between the mock and real api?

something like:

````ts
export const storageApi : IStorageApi = envConstants.isApiMock ? mockApi : api; 
```

Reply:
Adding @Nasdan to the loop fo my brainfart

Reply:
In other projects, we create this kind of interfaces like `storage.contract.ts`:

```
export interface StorageAPI {
....
}
```